ui/app/pages/swaps/fee-card/index.scss

```diff
@@ -61,6 +156,11 @@
     width: 10px;
     justify-content: center;
     margin-top: 2px;
+
+    div {
+      display: flex !important;
```

Comment:
Might be helpful to add a comment here as to why we're using `!important`, not sure if we have a standard for doing that with `important` usage.

Reply:
Done in 5d6c28567